src/components/App/App.tsx

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 import { Route, Routes } from 'react-router-dom';
 import { AppRootProps } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import FeatureFlagMissing from 'pages/FeatureFlagMissing';
 const Home = React.lazy(() => import('../../pages/Home'));
 
 export default function App(props: AppRootProps) {
+  let component = <Home />;
+  // TODO: Remove this once the feature flag is enabled by default
+  if (!(config.featureToggles as any)['grafanaAdvisor']) {
+    component = <FeatureFlagMissing />;
+  }
```

Comment:
I think you could also just return early?
```ts
if (!(config.featureToggles as any)['grafanaAdvisor']) {
  return <FeatureFlagMissing />;
}
```

(We don't really need the routes for now to be honest.)